mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/BrukerRoutes.kt.kt

```diff
@@ -37,11 +43,34 @@ fun Route.brukerRoutes() {
                 status = HttpStatusCode.BadRequest
             )
             val accessToken = call.getAccessToken()
-            historikkService.hentHistorikkForBruker(fnr, accessToken)?.let { call.respond(it) }
-                ?: call.respondText(
-                    "Klarte ikke hente historikk for bruker",
-                    status = HttpStatusCode.InternalServerError
-                )
+            auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' forsøker å se på tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+            historikkService.hentHistorikkForBruker(fnr, accessToken)?.let {
+                auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' har sett på tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+                call.respond(it)
+            }
+                ?: run {
+                    auditLog.log(createAuditMessage("NAV-ansatt med ident: '${getNavIdent()}' fikk ikke sett på tiltakshistorikken for bruker med ident: '${getNorskIdent()}'."))
+                    call.respondText(
+                        "Klarte ikke hente historikk for bruker",
+                        status = HttpStatusCode.InternalServerError
+                    )
+                }
         }
     }
 }
+
+private fun PipelineContext<Unit, ApplicationCall>.createAuditMessage(msg: String): CefMessage? {
```

Comment:
Jeg ser et potensiale for å trekke ut denne til en felles funksjon, men det kan hende det er litt prematurt å gjøre det. Det er feks. mulig å benytte en [miljøvariabelen](https://doc.nais.io/deployment/?h=environment#default-environment-variables) for applikasjonsnavnet.

Reply:
Ja, jeg var inne på tanken, men som du sier tror jeg det er litt prematurt. Jeg kan se om det er deler jeg kan trekke ut og benytte miljøvariabel istedenfor hardkodet streng uansett.